src/main/java/com/hierynomus/sshj/userauth/keyprovider/OpenSSHKeyV1KeyFile.java

```diff
@@ -133,6 +128,8 @@ protected KeyPair readKeyPair() throws IOException {
             }
         } catch (final GeneralSecurityException e) {
             throw new SSHRuntimeException("Read OpenSSH Version 1 Key failed", e);
+        } catch (Base64DecodeError e) {
+            throw new SSHRuntimeException("Corrupted private key: " + e.getMessage(), e);
```

Comment:
Is it necessary to include the exception message since it is already present in the stack trace?
```suggestion
            throw new SSHRuntimeException("Private Key decoding failed", e);
```

Reply:
> Is it necessary to include the exception message since it is already present in the stack trace?

We have some code that shows a dialog message with `getMessage()` when an absolutely unexpected error occurs. Surprisingly, something unexpected occurs quite often.

Anyway, it's not so important. I believe that it's not the library problem.

https://github.com/hierynomus/sshj/pull/936/commits/5c054bc71fe6ff5e759c3b95485b09bdef0543c2